src/attack_flow_builder/src/store/Commands/AppCommands/SavePageImageToDevice.ts

```diff
@@ -21,30 +15,25 @@ export class SavePageImageToDevice extends AppCommand {
      */
     constructor(context: ApplicationStore, id: string) {
         super(context);
```

Comment:
Same question about `id`. Is it still meaningful? Does it make sense to have it in the constructor signature?

Reply:
Again, good catch, it is not. Removed.